attributes/network/1000_abci_backend.adoc

```diff
@@ -16,5 +16,52 @@ It exposes the following endpoints:
 - `abci.endBlock`.
 - `abci.init`.
 - `abci.info`.
+- `abci.checkTx`.
+    Method controls what transactions get added to a new block
+- `abci.manyStatus`.
+    Includes nodes, sync and validator info.
+- `abci.validators`.
```

Comment:
I would be okay for `tendermint` specific APIs to just return a `tstr` that contains the JSON directly from tendermint.